virttest/storage.py

```diff
@@ -476,12 +476,10 @@ def is_disk_size_enough(required, available):
         """Check if available disk size is enough for the data copy."""
         minimum_disk_free = 1.2 * required
         if available < minimum_disk_free:
-            # bitshift trick to do a quick convert (MB) for the log
-            bitshift = 20
-            logging.error("Free space: %s MB", (available >> bitshift))
-            logging.error("Backup size: %s MB", (required >> bitshift))
+            logging.error("Free space: %s MB", (available / 1048576.))
+            logging.error("Backup size: %s MB", (required / 1048576.))
             logging.error("Minimum free space acceptable: %s MB",
-                          (minimum_disk_free >> bitshift))
```

Comment:
why not just use`int(minimum_disk_free)` here?

Reply:
either way is fine by me. It just doesn't seem worth optimizing as it only happens when there is not enough free space...

Reply:
@hereischen what do you think? My CI got interrupted because of this again...

Reply:
up to you

Reply:
well I'd like to get this merged, slightly prefer this version as it is more explicit and optimization is not really required.